INSTALL.md

```diff
@@ -64,7 +65,10 @@ There are several options to install these dependencies depending on your needs:
 - Alternatively, you could install them via conda. However, the conda repository may not have the exact versions of these dependencies that you want.
 - You could setup a full development environment with [Vagrant](https://www.vagrantup.com/), and continue using conda under it to primarily manage python dependencies. Note that we don't cover Vagrant in these instructions.
 
-### PostgreSQL installation on Mac OS X
+### PostgreSQL installation on Linux
+For Linux, you can install Postgres through these [instructions](https://computingforgeeks.com/how-to-install-postgresql-13-on-ubuntu/). These instructions were tested on Postgres v13 on Ubuntu v20.04.4.
```

Comment:
@aliu104 Thank you for adding this information. However, this link looks to be a blog post and as such, we can't rely on it to be available in the future. I would suggest paraphrasing their instructions and citing them as a source, or citing a webpage from say the PostgreSQL homepage.

Reply:
Done in draft two!